backend/tests/Feature/Modules/PantheonConnectorTest.php

```
@@ -0,0 +1,199 @@
+    }
+
+    private function settings(array $config, bool $active = true): PantheonSettings
+    {
+        return new PantheonSettings($config, $active);
+    }
+
+    // ── Finding 1: tenant isolation ─────────────────────────────────────────
+
+    public function test_pushing_documents_only_touches_the_active_tenants_documents(): void
+    {
+        $ours = Tenant::create(['name' => 'Our customer'])->id;
```

PHPStan (2.2.7)

[error] Call to an undefined static method App\Models\Tenant::create().


(staticMethod.notFound)